### PR TITLE
prevent mistake of releasing from wrong branch

### PR DESCRIPTION
**Pull request by patriknw**
To be picked to master also

### CODE REVIEW

File: project/scripts/release [hunk 1]
```diff
@@ -275,7 +275,11 @@ try ssh -t ${release_server} echo "Successfully contacted release server."
 
 echolog "Getting current project version from sbt..."
 declare -r current_version=$(get_current_version)
-echolog "Current version is ${current_version}"
+echolog "Current version is ${current_version} on branch $initial_branch"
+
+if [ ${current_version:0:3} != ${version:0:3} ]; then
```
Comment (ktoso) [inline project/scripts/release]: somewhat safer: `if [ "${current_version:0:3}" != "${version:0:3}" ];`

**Comment by ktoso** [comment]
LGTM, very good to include this right away.

File: project/scripts/release (hunk 1 above)
Comment (patriknw) [inline project/scripts/release]: ok, changed to that

**Comment by ktoso** [comment]
LGTM

**Pull request merged**